scripts/mutect2_wdl/mutect2-replicate-validation.wdl

```diff
@@ -76,10 +76,8 @@ workflow Mutect2ReplicateValidation {
 	File ref_dict
 	File? pon
 	File? pon_index
-	File? dbsnp
-	File? dbsnp_index
-	File? cosmic
-	File? cosmic_index
+	File? gnomad
```

Comment:
Why are we not adding gnomad?  Why can't we just include dbSNP, COSMIC, and gnomAD?